crates/bevy_pbr/src/render/pbr_prepass.wgsl

```diff
@@ -0,0 +1,149 @@
+#import bevy_pbr::prepass_bindings
+#import bevy_pbr::pbr_bindings
+
+// !!!
+// WARN this code is directly copied from pbr_functions.wgsl because of limitations with shader imports.
+// This is a temporary measure until better imports are supported.
+// !!!
```

Comment:
When I try importing the pbr_functions.wgsl files everything breaks because the bind groups are all completely different than what other functions in the file expects. The only solution would be to add _a lot_ of #ifdef to make it work correctly. I can still try it again, but the intentions was to keep this until we have better imports.

I can give it a try again and see if it's as bad as I remembered, but I think the issue is that pbr_functions also imports a lot of files  so I would need to wrap pretty much every function in a ifndef PREPASS

Reply:
Alright, well, after trying it out again, it seems like all I needed to do was wrap `pbr()` with a `#ifndef NORMAL_PREPASS`. We still need better imports, but this is definitely way better.